infrastructure/run_terraform.sh

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env bash
+ROOT=$(git rev-parse --show-toplevel)
+if ! sh "$ROOT/infrastructure/is_up_to_date.sh";
+then
+  exit 1
+fi
```

Comment:
Another way to do this (which we do in other scripts) would be to add

```shell
#!/usr/bin/env bash

set -o errexit
set -o nounset
```

as your header – fail the whole script if any individual line returns a non-zero exit code, fail if any variables are unset. How sensible programming languages behave!

Then you could just run `$ROTT/infrastructure/is_up_to_date.sh` on a line and it'll propagate the error code.

Reply:
Good point